src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -44,6 +58,7 @@ export const proxyRequest = ({
 }: Args) => {
   const { hostname, port, protocol, pathname, search } = uri;
   const client = uri.protocol === 'https:' ? https : http;
+  const encodedPath = encodePathname(pathname);
```

Comment:
could we add some tests to cover this new behaviour in https://github.com/elastic/kibana/blob/main/src/plugins/console/server/lib/proxy_request.test.ts ?